USER.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE "users" (
+  "userid" SERIAL PRIMARY KEY,
+  "password" varchar(255),
+  "firstname" varchar(255),
+  "lastname" varchar(255),
+  "phonenb" varchar(20),
+  "experience_level" varchar(255),
+  "education" varchar(255),
+  "description" varchar(255),
+  "job_role" varchar(255),
+  "joindate" timestamp default (now()),
+  "profile_picture" BYTEA default default_picture,
```

Comment:
we will not store the picture in the db, rather a url pointing to the storage. So a string would be enough. rename to profile_url